main.tf

```diff
@@ -22,7 +22,10 @@ resource "random_id" "random_role_id_suffix" {
 }
 
 locals {
-  base_role_id = "osLoginProjectGet"
+  base_role_id            = "osLoginProjectGet"
+  service_account_email   = var.service_account_email == "" ? google_service_account.bastion_host[0].email : var.service_account_email
+  service_account_project = var.service_account_email != "" ? split(".", split("@", "test@arc-friday.iam.gserviceaccount.com")[1])[0] : ""
```

Comment:
Fairly sure this was leftover from testing but we shouldn't need project right?

Reply:
The mistake here is that I leftover the service_account_email hardcoded for testing. But I do need the project to create the service_account_id

Reply:
actually, you're right... I don't need it anymore, sorry.

Reply:
done